src/main/webapp/app/modules/conflicts/all/all-records-view.tsx

```diff
@@ -43,6 +43,7 @@ export interface IAllRecordsViewState {
   fieldSettingsExpanded: boolean;
   tooltipOpen: boolean;
   selectedSettings: any;
+  loadingPartner: boolean;
```

Comment:
Why you are not handling this inside shared-record-view.reducer? You could then set loading to true, when you request fetch and to false when it succeeds or fails

Reply:
it's because we are not using reducers for dismissing or hiding records; it's worth refactoring, but not a part of this task

Reply:
https://github.com/benetech/ServiceNet/issues/1448

Reply:
Ok, sounds good.